tool/tbot/config/config_destination.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package config
+
+import (
+	"github.com/gravitational/trace"
+)
+
+type ConfigKind string
+
+const (
+	CONFIG_KIND_SSH ConfigKind = "ssh"
```

Comment:
```suggestion
	KindSSH ConfigKind = "ssh"
```

We're already in package `config`, so this should be sufficient.